include/chemfiles/formats/Molfile.hpp

```diff
@@ -90,11 +88,9 @@ class Molfile final: public Format {
 };
 
 template<> FormatInfo format_information<Molfile<DCD>>();
-template<> FormatInfo format_information<Molfile<GRO>>();
 template<> FormatInfo format_information<Molfile<TRR>>();
 template<> FormatInfo format_information<Molfile<XTC>>();
 template<> FormatInfo format_information<Molfile<TRJ>>();
-template<> FormatInfo format_information<Molfile<MOL2>>();
```

Comment:
I think the CI error is due to this not being updated in `FileFormat.cpp`

Reply:
Yes, sorry about this, I should have run the tests =/